spec/superspawn.spec.js

```diff
@@ -100,20 +125,20 @@ describe('spawn method', function () {
         let promise;
         expect(() => { promise = superspawn.spawn(TEST_SCRIPT, []); }).not.toThrow();
 
-        return Promise.resolve(promise).then(_ => {
+        return promise.then(() => {
             fail('Expected promise to be rejected');
-        }, err => {
+        }).catch(err => {
```

Comment:
This will catch your own `fail`

Reply:
Reverting this change then, as it is problematic.

```suggestion
        }, err => {
```

Reverting this change by itself does not seem to cause a test failure in my workarea.

- [ ] I will check that the build of this PR remains green.